doc/btp_spec.txt

```diff
@@ -1852,3 +1852,54 @@ Events:
 
 		This event indicates that segmented message incomplete timer
 		expired.
+
+	Opcode 0x89 - Friendship Established Event
```

Comment:
Would it be possible to have common Established and Terminated events for Friend and LPN with one additional Role parameter?

Reply:
It would be possible but I'm not sure if this is the way to go. How would we pass additional parameters that are specific to one role? And since this PR already has an LPN-specific event then I think it is cleaner to have separate events per role. I'm open to discussion :)

Reply:
I agree, let's keep those as separate commands as the arguments differ